### PR TITLE
Fixed non-Exception throwables being ignored in async tasks, and added a timed message

### LINKED ISSUE #1502: AsyncCommandBuilder drops Throwable/Errors
`AsyncCommandBuilder` only catches `Exception`, so any `Error` slips on by the error message system. This should not occur.

### PR DESCRIPTION
**Pull request by me4502**
Fixes #1502

### CODE REVIEW

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/util/AsyncCommandBuilder.java [hunk 1]
```diff
@@ -125,7 +133,12 @@ private AsyncCommandBuilder(Callable<T> callable, Actor sender) {
     public ListenableFuture<T> buildAndExec(ListeningExecutorService executor) {
         final ListenableFuture<T> future = checkNotNull(executor).submit(this::runTask);
         if (delayMessage != null) {
```
Comment (wizjany) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/util/AsyncCommandBuilder.java:135]: does this handle the case where delayMessage == null && repeatedMessage != null? what's intended behavior there? possibly it should use the repeatedMessage at 1 second and then repeat it every 10 as well?

Comment (me4502) [reply worldedit-core/src/main/java/com/sk89q/worldedit/command/util/AsyncCommandBuilder.java:135]: It currently doesn't, but IMO the delayMessage should be required to use the repeatedMessage. So I'll just make the builder require delayMessage when repeatedMessage is passed.

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/util/MessageTimerTask.java [hunk 2]
```diff
@@ -24,30 +24,41 @@
...
 
     @Override
     public void run() {
-        sender.printDebug(message);
+        if (!hasRunBefore) {
+            sender.printDebug(message);
+            hasRunBefore = true;
+        } else if (repeatedMessage != null) {
+            sender.printDebug(repeatedMessage);
+        }
```
Comment (octylFractal) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/util/MessageTimerTask.java:61]:
```suggestion
        } else {
            cancel();
        }
```
To save some CPU?

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/util/AsyncCommandBuilder.java [hunk 3]
```diff
@@ -94,6 +96,12 @@ private AsyncCommandBuilder(Callable<T> callable, Actor sender) {
         return this;
     }
 
+    public AsyncCommandBuilder<T> sendDelayedRepeatingMessage(Component message) {
```
Comment (octylFractal) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/util/AsyncCommandBuilder.java:99]: Shouldn't this be more like `sendMessageAfterFurtherDelay` (maybe `Delays`)? Currently this is in the present tense.

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java [hunk 4]
```diff
@@ -134,6 +134,7 @@ public void load(Actor actor, LocalSession session,
         AsyncCommandBuilder.wrap(task, actor)
                 .registerWithSupervisor(worldEdit.getSupervisor(), "Loading schematic " + filename)
                 .sendMessageAfterDelay(TranslatableComponent.of("worldedit.schematic.load.loading"))
+                .sendRepeatedMessage(TranslatableComponent.of("worldedit.schematic.load.still-loading"))
```
Comment (octylFractal) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java:137]: These didn't get changed, compile errors abound.

**Pull request merged**